src/pretix/control/forms/waitinglist.py

```diff
@@ -0,0 +1,22 @@
+from django_scopes.forms import SafeModelChoiceField
+
+from pretix.base.forms import I18nModelForm
+from pretix.base.models import WaitingListEntry
+
+
+class WaitingListEntryEditForm(I18nModelForm):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['subevent'].required = True
+        self.fields['subevent'].empty_label = None
+        self.fields['subevent'].queryset = self.event.subevents.filter(active=True)
+
+    class Meta:
+        model = WaitingListEntry
+        fields = [
+            'subevent',
+        ]
+        field_classes = {
+            'subevent': SafeModelChoiceField,
```

Comment:
We always use select2 for subevent choices, we should do so here as well

Reply:
I will add select2. In `typeahead.subevent_select2` I do not see a way to filter only active subevents – only options seems to filter by name and location. IMHO that is okay to also offer inactive events when transfering to subevents, as maybe one wants to offer a special subevent for waitinglist-users that is not active (yet).